resources/auth.py

```diff
@@ -17,3 +17,14 @@ def is_any_admin(user):
         is_general_admin(user) or
         user.unit_group_authorizations.admin_level().exists() or
         user.unit_authorizations.admin_level().exists())
+
+
+def is_unit_admin(user):
```

Comment:
It seems these are not used anywhere (`is_unit_admin` and `is_unit_manager`) after refactorings. Remove the dead code.